src/applications/mhv/secure-messaging/components/shared/SmBreadcrumbs.jsx

```diff
@@ -119,6 +151,15 @@ const SmBreadcrumbs = () => {
     ],

```

Comment:
dependency warning here (maybe you can just use `activeFolder` on its own? not sure)

Reply:
Thanks! I just pushed this fix. I ended up just using the active folder as you suggested.